scout/build/variant/variant.py

```diff
@@ -234,6 +234,10 @@ def build_variant(
     if "str_source" in variant:
         variant_obj["str_source"] = variant["str_source"]
 
+    # Mitochondria specific
+    if "mitomap_associated_diseases" in variant:
```

Comment:
From reading the code below, looks like this might be ".", why don't you check at this level and collect the value only if it's different from ".", so you avoid all the checks later in the templates?

Reply:
Good idea, fixed. I was annoyed with checking for the dot everywhere...